ClosedXML/Excel/XLWorksheet.cs

```diff
@@ -1491,5 +1492,19 @@ public Object Evaluate(String expression)
         }
 
         public String Author { get; set; }
+
+        public List<Drawings.IXLPicture> Pictures()
+        {
+            return pictures;
+        }
+
+        public void AddPicture(Drawings.XLPicture pic)

```

Comment:
so ⚪️  is nice to have, yea it be great if you can implment it, it would be nice, if we are smart when it comes to storing 

keep in mind, this can be tricky , identifying the "same picutre" , also please evaluate the cost of doing it